.github/workflows/ci.yml

```
@@ -32,9 +32,9 @@ jobs:
               uses: actions/checkout@v4
 
             - name: ⎔ Setup Node
-              uses: actions/setup-node@v4
+              uses: actions/setup-node@v6
```

Let's update these in a separate PR


```suggestion
              uses: actions/setup-node@v4
```